NOTICE

```diff
@@ -28,8 +22,9 @@ See bundled license files for details.
 This project bundles the following dependencies under SIL OFL 1.1 license (https://opensource.org/licenses/OFL-1.1).
 See bundled license files for details.
 
-- font-awesome:4.5.0 (Font) (http://fortawesome.github.io/Font-Awesome/) - Created by Dave Gandy
-    -> fonts in "docs/page/font-awesome/fonts"
+- font-awesome:4.6.3 (Font) (https://fontawesome.com/) - Created by Dave Gandy
```

Comment:
How did you arrive at this version?

Reply:
Hello Chesnay, I found the css version number to be 4.6.3 here.
https://github.com/apache/flink/blob/master/docs/static/font-awesome/css/font-awesome.css#L2
I am not sure if there is a version for fonts.

Reply:
lets assume that the version applies to both